pkgs/applications/audio/furnace/default.nix

```diff
@@ -104,9 +103,7 @@ stdenv.mkDerivation rec {
   '';
 
   passthru = {
-    updateScript = gitUpdater {
-      rev-prefix = "v";
-    };
+    updateScript = ./update.sh;
```

Comment:
```suggestion
    updateScript = gitUpdater {
      rev-prefix = "v";
      ignoredVersions = "pre";
    };
```

Reply:
We just bumped from a pre-release to a full release (#259128), after bumping from pre-release to pre-release for awhile. I'm not sure this "year-and-a-half of big pre-releases" situation will never repeat now that it has happened once already, and I'd rather not disregard all updates for several months if it does.